src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/OOPTagHelperResolver.cs

```diff
@@ -104,24 +104,21 @@ protected virtual async Task<TagHelperResolutionResult> ResolveTagHelpersOutOfPr
             // This will change in the future to an easier to consume API but for VS RTM this is what we have.
             try
             {
-                var client = await RazorLanguageServiceClientFactory.CreateAsync(_workspace, CancellationToken.None).ConfigureAwait(false);
-                if (client != null)
+                var remoteClient = await RazorRemoteHostClient.CreateAsync(workspaceProject.Solution.Workspace, CancellationToken.None);
```

Comment:
>workspaceProject.Solution.Workspace [](start = 75, length = 35)

Can we instead use `_workspace` here? We plan to obsolete `Solution.Workspace` in future.